source/common/common/thread.h

```diff
@@ -57,26 +58,40 @@ class MutexBasicLockable : public BasicLockable {
  */
 class CondVar {
 public:
-  // Note that it is not necessary to be holding an associated mutex to call
-  // notifyOne or notifyAll. See the discussion in
-  //     http://en.cppreference.com/w/cpp/thread/condition_variable_any/notify_one
-  // for more details.
+  enum WaitStatus {
```

Comment:
enum class, please

Reply:
I was alright with referencing the status enum values under CondVar, but another indirection is OK too; done.